cmd/ctr/utils.go

```diff
@@ -79,3 +83,29 @@ func waitContainer(events execution.ContainerService_EventsClient, id string, pi
 		}
 	}
 }
+
+// ForwardInterruptSignal forwards the ctrl+c (interrupt) signal  to stop all container processes when exiting.
+func ForwardInterruptSignal(containers execution.ContainerServiceClient, id string) chan os.Signal {
+	sigc := make(chan os.Signal, 1)
+	signal.Notify(sigc, os.Interrupt)
+	go func() {
+		for range sigc {
```

Comment:
Instead of just handling one signal you can set this up to forward all signals.  Sorry for not being clear in my issue, its less about handling ctrl+c specifically, but forwarding all signals from the client to the container.